akka-actor/src/main/mima-filters/2.5.14.backwards.excludes

```diff
@@ -0,0 +1,17 @@
+# #25340 ReceiveTimeout
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.actor.ActorCell.checkReceiveTimeout")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.actor.dungeon.ReceiveTimeout.checkReceiveTimeout")
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.actor.dungeon.ReceiveTimeout.checkReceiveTimeout$default$1")
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.actor.dungeon.ReceiveTimeout.checkReceiveTimeout")
+
+# Additional records support for async dns, DnsProtocol is marked with ApiMayChange
```

Comment:
@jrudolph added the ApiMayChange comment here
I think a new release of akka-management is planned anyway, using these new records, https://github.com/akka/akka-management/pull/275

Reply:
Ok, I guess it's fine that akka-management is broken until then. Nothing we can do here anyway.